### ISSUE TITLE
Do not generate controller - what I am doing wrong?

### ISSUE THREAD

**Issue by ignatandrei** [cloud0259/ControllerGenerator#10]
I have a list of Roslyn Source Code Generator with demos ( https://ignatandrei.github.io/RSCG_Examples/v2/docs/List-of-RSCG) and I have tried to add yours 

The example is on the branch https://github.com/ignatandrei/RSCG_Examples/tree/105-controllergenerator .
The solution is at <RSCG_Examples>\v2\rscg_examples\ControllerGenerator\src\ControllerGeneratorDemo.sln

I have put 
```xml
 <PropertyGroup>
        <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
        <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GX</CompilerGeneratedFilesOutputPath>
    </PropertyGroup>
```
but it does not generate the controller.

Could you please advise?

**Comment by cloud0259** [comment]
Hello

The issue arises from the number of characters in the file path. By moving the project to the root of the C:/ drive, the problem is resolved. 

![image](https://github.com/cloud0259/ControllerGenerator/assets/6765644/18b0b3f1-b4bd-4ad8-a418-8363007fc451)

Windows restricts the number of characters in a file path to 250 characters. This limitation can be removed:

https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry"

**Comment by ignatandrei** [comment]
This is very curious. I have put into the root folder, but still did not generate controllers. 
![image](https://github.com/cloud0259/ControllerGenerator/assets/153982/71834e64-8bec-44c9-ba9d-aebefc1be6a2)

Do you use my csproj with 

    <PackageReference Include="ControllerGenerator" Version="1.3.2" OutputItemType="Analyzer" />

or did you put the reference to your csproj ?

**Comment by ignatandrei** [comment]
Or maybe because the 2 packages have different versions (3.0 vs 3.2)?

PackageReference Include="ControllerGenerator.Abstraction" Version="**1.3.0**"

PackageReference Include="ControllerGenerator" Version="**1.3.2**"

**Comment by cloud0259** [comment]
I just tried again with your project and it works very well.
[ControllerGenerator.zip](https://github.com/cloud0259/ControllerGenerator/files/12821670/ControllerGenerator.zip)

Here is the copy of your project. I haven't changed anything except putting it at the root of my disk.

**Comment by ignatandrei** [comment]
The namespace is different!

You have put
```csharp
namespace BLL.Services;
public class EmployeeLogic : IEmployeeLogic
//and for all csharp files in the project BLL
```

While I do have
```csharp
namespace BLL;
public interface IEmployeeLogic: IAutoGenerateController
//and for other classes
```


Do you hardcode somewhere .Services ? ;-)

Anyway , I discovered the problem and reproduced. If I put the 

namespace BLL.Services;

it works. But it should work with either namespace ;-)

**Comment by cloud0259** [comment]
Indeed, there must be a bug on the search for services. I'll fix this bug when I have a moment. Thanks for the feedback.

**Comment by ignatandrei** [comment]
Did you have the time to work ?

**Comment by ignatandrei** [comment]
?

**Comment by ignatandrei** [comment]
?

**Comment by cloud0259** [comment]
Hello, sorry, I don't have time to fix this right now.

**Comment by ignatandrei** [comment]
Maybe .... now?
